src/utils/logging_config.toml

```diff
@@ -8,10 +8,10 @@ file_mode = "w+"
 set_global = true
 
 ["group_levels"]
-# Set these to Info to suppress the group when debug logging is enabled.
+# Uncomment these to suppress the group when debug logging is enabled.
```

Comment:
Can you entries to the documentation about this in the PR you have some time? I think this is useful but we need to have a good explanation of this for the users (including us) to actually exploit it.

Reply:
Did you see these updates in the last PR?
- https://nrel-siip.github.io/InfrastructureSystems.jl/stable/dev_guide/logging/#Use-Cases
- https://nrel-siip.github.io/InfrastructureSystems.jl/stable/dev_guide/tests/

Reply:
Yes, I meant on how to modify the TOML file to achieve certain behaviors

Reply:
I added more documentation about what we are doing with these groups but am not sure I addressed your concerns. Aren't the comments in this file self-explanatory?